onnxruntime/core/providers/cuda/nn/conv.cc

```diff
@@ -10,10 +10,18 @@ namespace onnxruntime {
 namespace cuda {
 
 #define REGISTER_KERNEL_TYPED(T)                                                \
+  ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_EX(                                      \
+      Conv,                                                                     \
+      kOnnxDomain,                                                              \
+      1, 10,                                                                    \
+      T,                                                                        \
+      kCudaExecutionProvider,                                                   \
+      KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
+      Conv<T>);                                                                 \
   ONNX_OPERATOR_TYPED_KERNEL_EX(                                                \
```

Comment:
Please add a comment explaining why the opset 11 implementation can use the same code